.github/workflows/publish-docker-images.yaml

```diff
@@ -0,0 +1,81 @@
+name: Publish Docker Images
+
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    # The branches below must be a subset of the branches above.
+    branches: [main, master]
+  release:
+    types: [published]
+
+jobs:
+  publish_copilot_proxy:
+    name: Publish copilot_proxy
+    runs-on: ubuntu-latest
```

Comment:
Latest? Which better between latest version and specified version (e.g., 22.04, 20.04, ) for maintenance and test?

Reply:
Pinning would make it more robust, but given that this is just ubuntu, I think we can survive with latest.